src/components/TopicLoader/TopicLoader.js

```diff
@@ -97,7 +97,7 @@ class TopicLoader extends Component {
       }
       this.updateLayers(activeTopic.layers);
 
-      searchService.setSearches(activeTopic.searches);
+      searchService.setSearches(activeTopic.searches || []);
```

Comment:
```suggestion
      searchService.setSearches(activeTopic.searches || {});
```
Not sure if this makes a difference, but searchService expects an object for searches.